README.md

```diff
@@ -12,61 +13,116 @@ To install run
 composer require itk-dev/openid-connect
 ```
 
+If you wish to run the coding standard tests for Markdown files
+
+```sh
+yarn add markdownlint-cli --dev
```

Comment:
This could be replaced by `yarn install` as you already have a package.json and yarn.lock file present with markdownlint defined as a dependency.